sklego/loess.py

```diff
@@ -0,0 +1,332 @@
+import logging
+from math import floor
+
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.base import BaseEstimator
+from sklearn.base import TransformerMixin
+from sklearn.linear_model import LinearRegression
+from sklearn.neighbors import NearestNeighbors
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import PolynomialFeatures
+
```

Comment:
`Loess` can be part of `sklego.linear`.

Reply:
Considering:
- Loess can use non-linear models for the local fits
- The local model fitting is non-linear in itself

Are you sure about this?

Reply:
actually ... we need to follow the sklearn api so it should be `sklego.linear_model` 

the local model *is* linear, it might just have polynomial features going in. the *features* are non-linear such that the model can be.